src/reactjs/useSyncExternalStore.native.ts

```diff
@@ -0,0 +1,4 @@
+// @ts-ignore
+// eslint-disable-next-line import/no-unresolved
+import { useSyncExternalStore } from 'use-sync-external-store/shim/index.native.js'
+export { useSyncExternalStore }
```

Comment:
could we also do:

```
export { useSyncExternalStore } from 'use-sync-external-store/shim/index.native.js'
```

Reply:
I just copied the style it was done here:
https://github.com/tannerlinsley/react-query/blob/5f80b0906b5b342782af222f75bbf78d6d267151/src/reactjs/reactBatchedUpdates.native.ts

Reply:
fixed